INSTALL.md

```diff
@@ -7,92 +7,123 @@ which includes this INSTALL.md file, is denoted $(CERTIFIER).
 The Ubuntu 20.04 install guide can be found in
 [Ubuntu Install](./Doc/install-certifier-Ubuntu-20.04.md).
 
-Read the Guide in the Doc directory to understand the basic nomenclature
-and programs you will build and use.
+Read the [Guide](Doc/Guide.pdf) in the Doc directory to understand the
```

Comment:
Many changes are of this kind, adding a hyper-link xref to the section being referenced. This helps one to quickly navigate to the referenced file / sub-dir,  and have a quick look-see.